custom_random_forest.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import random
+from concurrent.futures import ProcessPoolExecutor
+
+from sklearn import datasets
+from sklearn.base import BaseEstimator
+from sklearn.tree import (DecisionTreeRegressor,
+                          DecisionTreeClassifier)
+from sklearn.model_selection import train_test_split
+from sklearn.datasets import make_classification
+
+
+class RandomForestClassifierCustom(BaseEstimator):
+    def __init__(
+        self, n_estimators=15, max_depth=9, max_features=2, random_state=111
+    ):
+        self.n_estimators = n_estimators
+        self.max_depth = max_depth
+        self.max_features = max_features
+        self.random_state = random_state
+
+        self.trees = []
+        self.feat_ids_by_tree = []
+
+    def creating_tree(self, tup):
+        X, y, i = tup
+        random_state = np.random.seed(self.random_state + i)
```

Comment:
По сути с функцией все супер. Пару советов по улучшению:
1) Название функции наверное лучше было бы не `creating_tree` а например `fit_tree`. И я бы еще нижнее подчеркивание добавил, чтобы показать что это не часть публичного интерфейса (`_fit_tree`)
2) `i` это же не просто немой счетчик. Я бы назвал это например `tree_id`
3) Ну и `tup` тоже немного странное имя)
```suggestion
    def _fit_tree(self, args):
        X, y, tree_id = args
        random_state = np.random.seed(self.random_state + tree_id )
```